app/models/manageiq/providers/microsoft/infra_manager/refresh_parser.rb

```diff
@@ -491,7 +491,7 @@ def create_relationship_tree
 
       host_folder = {
         :name          => 'host',
-        :is_datacenter => false,
+        :type          => EmsFolder.name,
```

Comment:
You can just hardcode the string `"EmsFolder"`...not sure it's worth the overhead to call a method, and I think it's consistent within the parsers to just use the String.